events-collector/events-collector-api/src/types/events.rs

```diff
@@ -0,0 +1,127 @@
+//! Event types collected by the events collector
+
+use alloy_primitives::{Address, U256};
+use renegade_circuit_types::order::OrderSide;
+use renegade_common::types::exchange::Exchange;
+use serde::{Deserialize, Serialize};
+use uuid::Uuid;
+
+/// Wallet creation event
+#[derive(Serialize, Deserialize)]
+pub struct WalletCreation {
+    /// The wallet ID
+    pub wallet_id: Uuid,
+}
+
+/// Deposit event
+#[derive(Serialize, Deserialize)]
+pub struct Deposit {
+    /// The wallet ID
+    pub wallet_id: Uuid,
+    /// The mint of the asset being deposited
+    pub mint: Address,
+    /// The amount of the deposit
+    pub amount: U256,
```

Comment:
We only allow `u128` amounts throughout the system (same for the amount fields in the order placement, withdraw, etc). So i'd change this to `u128`